src/lib/components/workflow/workflow-summary.svelte

```diff
@@ -31,16 +31,14 @@
       $workflowSummaryViewOpen = !$workflowSummaryViewOpen;
     }}
   >
-    <div
-      class="grid-row-3 grid w-full grid-cols-1 gap-4 md:grid-cols-2 lg:grid-cols-4 xl:w-11/12"
-    >
-      <div class="col-span-1 md:col-span-2">
+    <div class="flex flex-col gap-2 xl:flex-row">
+      <div class="grow">
```

Comment:
Unfortunately, this breaks truncation 😞 

![Screenshot 2023-11-03 at 3 58 26 PM](https://github.com/temporalio/ui/assets/15069288/c4aeaeb1-3284-4887-9232-525c6d4d8bfe)

Reply:
👍